.travis.yml

```diff
@@ -42,7 +44,6 @@ env:
 matrix:
   allow_failures:
     - env: ROS_DISTRO=indigo PRERELEASE=true PRERELEASE_REPONAME=industrial_core PRERELEASE_DOWNSTREAM_DEPTH=0
-    - env: ROS_DISTRO=indigo PRERELEASE=true PRERELEASE_REPONAME=ros_canopen
```

Comment:
I removed the first line only recently ;)
Instead of testing `ros_canopen` we might test something more stable like `std_msgs`.
I doesn't really matter which package to prerelase-test in order to test our scripts.